ui/src/components/ort-run-job-status.tsx

```diff
@@ -33,20 +35,34 @@ import { RunDuration } from './run-duration';
 type OrtRunJobStatusProps = {
   jobs: PagedResponse_OrtRunSummary_['data'][0]['jobs'];
   pollInterval: number;
+  orgId: string;
+  productId: string;
+  repoId: string;
+  runIndex: string;
 };
 
 export const OrtRunJobStatus = ({
   jobs,
   pollInterval,
+  orgId,
+  productId,
+  repoId,
+  runIndex,
 }: OrtRunJobStatusProps) => {
   return (
     <TooltipProvider>
       <div className='flex items-center space-x-1'>
         <Tooltip>
           <TooltipTrigger asChild>
-            <div
-              className={`${getStatusClass(jobs.analyzer?.status)} ${getStatusBackgroundColor(jobs.analyzer?.status)}`}
-            ></div>
+            <Link
+              to={`/organizations/${orgId}/products/${productId}/repositories/${repoId}/runs/${runIndex}/config#analyzer`}
+            >
```

Comment:
To benefit from TanStack Router's type safety, this and other links should use `to`, `params` and `hash`.

https://tanstack.com/router/latest/docs/framework/react/guide/navigation#dynamic-links
https://tanstack.com/router/latest/docs/framework/react/guide/navigation#hash-links

Reply:
Thanks, `hash` was what I was missing!